src/muiThemeable.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
 export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+  function MuiComponent(props, {muiTheme = getMuiTheme()}) {
```

Comment:
Absolutely! :+1: 

I think there's a lot we can benefit from in this iteration's implementation and newer more invasive approaches (such as the one discussed in #2509) should be discussed and considered carefully.